stable/etcd-operator/templates/operator-deployment.yaml

```diff
@@ -41,6 +45,9 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
+        ports:
```

Comment:
Why are we opening this port?

Reply:
so prometheus can scrape it..

Reply:
OK well a few things. I would probably want this in another PR to begin. The main concern here is that via commandArgs one can change that port value to something else, i believe that port is configurable and the default 8080 is used but this is hard coded so maybe also expose this as a configurable value. Once we have that figured out I think we should/would want want to be able to enable/disable metrics via the values file and create a clusterIP service type so that it is accessible to the cluster.  What do you think?

Reply:
about customizing the port I agree - we are hard coding the port on liveness/readyness probes as well, so it won't even start as is... 

about the service... we can do that, it's required if you're using prometheus-operator + servicemonitors, for the "standalone prometheus" we don't really need it though...

that being said, I can work on the port config + service on another PR if you want...